### DISCUSSION TITLE
Avast keeps blocking ImageGlass 3.2.0.16

### DISCUSSION THREAD

**Discussion by HanatakeYurii** [d2phap/ImageGlass#70] [General] [answered]
Hi

Don't know why but Avast keeps blocking ImageGlass last version reporting false positive:
Win32-Evo-Gen [Susp](I suppose Susp stands for Suspicious)

I don't know if can you can do anything, but I though useful to report.

**Comment by d2phap** [comment]
Hi @HanatakeYurii 

Thank you for your information. I have reported suspected false positive to Avast already.
You can disable Avast shield for a while to download it.

Phap

**Comment by HanatakeYurii** [comment]
Hi,

Yes I disabled it. I reported it as well for the installation (yes it was blocked again). Had to add it at ignore lists (I mean all of them, Avast would always find a way to block it if I didn't).

Hope they will fix this.

Hanatake

**Comment by Levan7** [comment]
Same issue here

**Comment by d2phap** [comment, marked as answer]
Just got email from Avast.

|
|
|

> Hello,
> 
> Thank you for contacting Avast Software.
> 
> The new virus definition is not flagging the files you submitted. I'd recommend updating Avast virus definitions and the program version. 
> Open Avast and go to > SETTINGS > Update 
> a) Virus Definition -> click 'Update'
> b) Program -> click 'Update'
> 
> In case if the file is still flagged, please send us the file(s) which is/are flagged as a password-protected compressed file (e.g. ZIP file, RAR file etc.). Please, compress the file(s) using the password "infected".
